Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -1176,7 +1176,7 @@ public final class PackageBuilder {
                     throw InternalError("only Swift supports swift language version")
 
                 case .swift:
-                    decl = .OTHER_SWIFT_FLAGS
+                    decl = .SWIFT_VERSION
                 }
 
                 values = ["-swift-version", version.rawValue]
```

Comment:
This should not have `-swift-version` here.

Reply:
Done!